### PR TITLE
Feature explore page #11

### PR DESCRIPTION
**Pull request by Tojonirina4**
Explorer page completed.
I'm not sure about the font-family used in the mockup.
@sina-jamshidi, Please let me know in case you know the font-family name. Thank you :)

### CODE REVIEW

**Comment by Tojonirina4** [comment]
ExplorerUi

**Review by johnsciutto — APPROVED** [review]

File: client/src/pages/explorer.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    width: "5px",
+    height: "5px",
+  },
+});
+
+const CustomSmallerCheckBox = withStyles({
+  root: {
+    "& .MuiSvgIcon-root": { width: "15px", height: "15px" },
+  },
+})(Checkbox);
```
Comment (sina-jamshidi) [inline client/src/pages/explorer.js]: let's bring this style to theme.js

File: client/src/pages/explorer.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+              }}
+            >
+              <div
+                style={{
+                  display: "flex",
+                  height: "15%",
+                  borderTop: "1px solid #a9a9a9",
+                  padding: 10,
+                  justifyContent: "space-between",
+                }}
```
Comment (sina-jamshidi) [inline client/src/pages/explorer.js]: We can move a lot of these up to the `makeStyles`. You can use the [classNames](https://www.npmjs.com/package/classnames) package to combine multiple styles if you need.

**Review by sina-jamshidi — APPROVED** [review]
looks good!

**Pull request merged**